packages/database/src/migrations/20230314042110-MigrateInsertNumberOfFacilitiesColumn-modifies-data.js

```diff
@@ -144,7 +144,7 @@ const updateStandardReports = async db => {
             transform: 'fetchData',
             dataTableCode: 'entity_relations',
             parameters: {
-              entityCodes: '=@all.organisationUnit',
+              entityCodes: '=@all.organisationUnit or []',
```

Comment:
The other option would have been to change the behaviour of the @all selector to return [] instead of undefined when there are no matching rows. I went with this option for quickness.